src/content-script.ts

```diff
@@ -4,14 +4,47 @@ import Mustache from "mustache";
 import { createElement } from "./elements";
 import { isBodyWithReplies, isRepositoryDetails } from "./validations";
 
-(async function main() {
+const regexPathName = /\/.+\/.+\/(issues|pull)\/\d+/;
+const replySummaryId = "saved-reply-new_comment_field";
+const footprint = "refined-saved-replies";
+
+// hack for detecting page change
+function addObserver(callback: MutationCallback) {
+	const options = { childList: true, subtree: true };
+
+	const observer = new MutationObserver(callback);
+
+	observer.observe(document.body, options);
+}
+
+function verifyLocation() {
```

Comment:
[Naming] This could be a little more clear - it's not (just) that we _verify_ the location; it's also that we run `main` if it matches.

How about...

```suggestion
function runMainOnLocationMatch() {
```

I'm not super happy about that name, if you have a cleaner one that'd be great 😄 

`runMainOnLocationMatch`? `runMainIfLocationMatches`? ...